app/src/main/java/com/restaurantroulette/service/CountryService.kt

```diff
@@ -21,8 +21,6 @@ class CountryService {
              * exception occurred creating the request or processing the response.
              */
             override fun onFailure(call: Call<ArrayList<Country>>, t: Throwable) {
-                val j = 1 + 1
```

Comment:
Removing this is good, as it is not adding value... but replace it with something useful, like a logging statement.